app-crypt/tpm2-tools/files/tpm2-tools-5.1.1-no-efivar-automagic.patch

```diff
@@ -0,0 +1,32 @@
+From 0cea7f0f78f1a9e8dca789eb5f2ece052e026bed Mon Sep 17 00:00:00 2001
```

Comment:
1. Upstreamable? If so, chuck in a link.
2. Always good to chuck in a link to the bug anyway.